### DISCUSSION TITLE
Is there a way to disallow drag on a single gridlist item?

### DISCUSSION THREAD

**Discussion by GermanJablo** [adobe/react-spectrum#5697] [Q&A] [answered]
What the title says.
I saw the `disabled Behavior` property, but if I disable drag for an item I also disable selection, and I don't want that.

**Comment by reidbarber** [comment]
You can filter it out in the `getItems` function you pass into `useDragAndDrop `.

**Reply by GermanJablo** [reply]
Yes, I forgot to say that that was the first thing I tried.
Although I get the dragEvent to have no content or data to drag, I can still drag the item.
In other words, `data-drop-target`, `react-aria-DropIndicator` and `renderDragPreview` still appear.

I know that in `renderDragPreview` I could return an empty div for example, but the mouse would still show that something is being dragged, and the drop-target/indicator still activates.

Is there something I'm missing?

**Reply by GermanJablo** [reply]
Also, a little context about my use case, because I think it may be helpful.

I'm trying to fake a Tree with a gridlist (turning the nesting level into an indent).

![image](https://github.com/adobe/react-spectrum/assets/43938777/2e0a8a8b-c9a4-46a5-bcd8-e39cceaed382)

The Home element has some peculiarities:

- ❌ I don't want it to be draggable.
- ❌ I don't want to be able to drop "before" or "after" it (`onReorder`).
- ✅ I want it to be able to drop "inside" it (`onItemDrop`).

**Reply by reidbarber** [reply, marked as answer]
I believe this has been discussed before, but I couldn't find it anywhere. I'm not sure that we have a solution for this at the moment. I think it is worth opening an issue for though, so feel free to do that. In the meantime, I don't think  I have a good solution.

**Reply by GermanJablo** [reply]
Thanks, done: https://github.com/adobe/react-spectrum/issues/5704
